spec/controllers/items_controller_spec.rb

```diff
@@ -331,15 +299,16 @@
     end
   end
   describe 'resource' do
-    it 'should set the object and datastream, then call the view' do
-      expect(Dor::Item).to receive(:find)
-      allow(@item).to receive(:datastreams).and_return({'contentMetadata' => double(Dor::ContentMetadataDS)})
-      get 'resource', :id => @pid, :resource => '0001'
-      # XXX : isn't actually testing what it says!
-    end
+    it 'should set the object and datastream, then call the view'
+    #   expect(Dor::Item).to receive(:find)
+    #   allow(@item).to receive(:datastreams).and_return({'contentMetadata' => double(Dor::ContentMetadataDS)})
+    #   get 'resource', :id => @pid, :resource => '0001'
+    #   # XXX : isn't actually testing what it says!
+    # end
```

Comment:
Well if ` # XXX : isn't actually testing what it says!` is true, make it pending!